src/PortlandLabs/ConcreteCmsTheme/User/CommunityUser.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace PortlandLabs\ConcreteCmsTheme\User;
+
+use Concrete\Core\Entity\User\User;
+use Concrete\Core\User\Avatar\AvatarInterface;
+use PortlandLabs\ConcreteCmsTheme\User\EmptyAvatar;
+use League\Url\Url;
+
+class CommunityUser
```

Comment:
I didn't see this class get used anywhere, is it used in such a way that `CommunityUserInspector` can't just be a dependency?